go.mod

```diff
@@ -251,6 +247,22 @@ require (
 	gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d // indirect
 )
 
+require (
```

Comment:
Why is there a separate require section here? Did you follow the instructions [here](https://github.com/grafana/grafana/blob/main/UPGRADING_DEPENDENCIES.md#go-dependencies)?

Reply:
I did `go mod tidy` and it was the result. I'm going to check why it add this at this way :/